provisions/roles/db/templates/cccp_db_pg_dump_cron.sh.j2

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+BACKUPDIR="{{ db_backup_nfs_path}}"
+
+DBPORT="{{ db_port }}"
+DBUSER="{{ db_user }}"
+DBNAME="{{ db_name }}"
+DBPASS="{{ db_pass }}"
+
+PGIMAGE="{{ postgresql_image }}"
+
+DATE=`date +%Y-%m-%d-%H-%M-%S`
+OUTPUTFILE="/tmp/$DATE-$DBNAME-pgdump.tar"
+
+PGDUMPCOMMAND="PGHOST=\$POSTGRES_PORT_5432_TCP_ADDR PGPORT=$DBPORT PGDATABASE=$DBNAME PGUSER=$DBUSER PGPASSWORD=$DBPASS pg_dump -F tar -v -d $DBNAME > $OUTPUTFILE"
+
+function pg_backup_database
+{
+  docker run --rm --link postgres -v $BACKUPDIR:/tmp $PGIMAGE bash -c "$PGDUMPCOMMAND"
+  echo "finished."
+}
+
+echo "------------------------------------------------------"
+echo `date +'%F %T'` Starting postgresql $DBNAME db backup..
+echo "------------------------------------------------------"
+pg_backup_database
```

Comment:
could you please point me some reference of `pg_backup_database`

Reply:
@bamachrn What do you mean by reference of pg_backup_database? It's a bash function defined in same file at line no:16